src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -419,6 +437,29 @@ private UserSettings LoadSettings()
             return settings;
         }
 
+        private void AddPackagesErrorBar()
```

Comment:
can't we just toggle visibility of the packages error bar? 

setting positions from code and inserting elements into the visual tree seems shady.